app/models/user.php

```diff
@@ -122,6 +122,40 @@ class User extends AppModel
         )
     );
 
+    private $defaultSettings = array(
+        'is_public' => false,
+        'lang' => null,
+        'use_most_recent_list' => false,
+        'collapsible_translations' => false,
+        'restrict_search_langs' => false,
+    );
+
+    public function afterFind($results, $primary = false) {
+        foreach ($results as &$result) {
+            if (isset($result['User']) && array_key_exists('settings', $result['User'])) {
```

Comment:
I’m unable to reproduce this bug. I don’t get it at all. It has to do with the latests comments displayed on the home page. Is there anything special about these?